src/api/graphQL/tokens/types.ts

```diff
@@ -12,11 +12,9 @@ export interface Token {
   token_id: number
   collection_id: number
   collection_name: string
-  data: {
-    image: {
-      url?: string;
-      fullUrl?: string;
-    }
+  image: {
+    url?: string;
+    fullUrl: string | null;
```

Comment:
Lost ipfsCid and urlInfix.

But maybe it not needed for frontend.

Reply:
I understood that if we are a scan that just displays the info, then fullUrl is enough for us. the other fields are needed to encode the token.  Do we have a need at the front to encode the token?